packages/docker-cli/Cargo.toml

```diff
@@ -9,8 +9,8 @@ build = "build.rs"
 path = "pkg.rs"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/docker/cli/archive/v19.03.15/cli-19.03.15.tar.gz"
-sha512 = "163f67a11b1d976eb91c84e59eb1fabf3f26b360ac726bb0bedb9a1532ef807b665f7461f52f6b236cb8af955af042466070800cd4d09b7a6b90092daf718b41"
+url = "https://github.com/docker/cli/archive/refs/tags/v20.10.4.tar.gz"
```

Comment:
Just curious, is there a difference between getting the source files from `https://github.com/docker/cli/archive/v"${VERSION}"/cli-"${VERSION}".tar.gz` and `https://github.com/docker/cli/archive/refs/tags/v"${VERSION}".tar.gz`?
I'm guessing the former is downloading from a release tag, and the latter is downloading from a tag?

Reply:
I had a typo when I tried to download from archive instead of ref/tag, and I thought there wasn't any 20.10.4 version available through the archive. I was using 20.10.04 instead of 20.10.4, which is what I use now.